packages/@aws-cdk/aws-redshift-alpha/test/integ.database.ts

```diff
@@ -47,7 +47,7 @@ const databaseOptions = {
 const user = new redshift.User(stack, 'User', databaseOptions);
 const table = new redshift.Table(stack, 'Table', {
   ...databaseOptions,
-  tableName: 'my_table',
+  tableName: 'IntegTable',
```

Comment:
What was the reasoning on this change?

Reply:
To force an update request in the integ test, which will update the table name

Reply:
So the integ tests are not persistent resources. It deploys and tears down the resources each time, so this update will not actually trigger the update request.

Have you tested this locally? Can you provide confirmation (possibly with screenshots) that the name update works as intended?

Reply:
Is there a specific way to test this locally? Is there a way for me to extract the build and perform my own test on a personal CDK app?

Reply:
@Rizxcviii, yes! The way I do it is by making a new app with `cdk init`. Then from the base of that app, you can use the linkall command: `~ ./../../<path-to-your-cdk-fork>/link-all.sh`. This should symlink your `node_modules` folder to be that of your current build in your fork. Then deploying from there normally.

Reply:
I've finished the necessary tests, it did come with additional changes. The screenshots are in a separate comment below